<launch>
  <arg name="quad_name" default="hummingbird"/>
  <group ns="rrt_planner_server">
    <rosparam file="$(find rrt_planner)/config/rrt_params.yaml" />
  </group>

  <node pkg="rrt_planner" type="rrt_planner_server" name="rrt_planner_server" output="screen"> 
    <param name="map_path" value="$(find maps)/config/$(arg obs_config)"/>
  </node>

  <group ns="$(arg quad_name)" >
    <node pkg="local_planner" type="local_planner" name="local_planner" output="screen" />
  </group>

</launch>
